docs/release-notes/release-notes-0.18.0.md

```diff
@@ -330,6 +332,10 @@ bitcoin peers' feefilter values into account](https://github.com/lightningnetwor
 * The [`estimateroutefee`](https://github.com/lightningnetwork/lnd/pull/8136)
   subcommand now gives access to graph based and payment probe fee estimation.
 
+* [`sendcoins` now takes in an optional utxo flag](
+ https://github.com/lightningnetwork/lnd/pull/8516). This allows users to
```

Comment:
I think this will not render properly due to a limitation of the Markdown parser. I think you need to break before the word "flag" instead.

Reply:
Or, maybe that used to be the case and was fixed in the meantime... Looks fine here: https://github.com/lightningnetwork/lnd/blob/41c884aa28c317db34b8d80350f79cca7b9fe33d/docs/release-notes/release-notes-0.18.0.md#lncli-updates
So feel free to ignore.